src/app/src/components/FilterSidebarFacilitiesTab.jsx

```diff
@@ -317,11 +317,11 @@ function FilterSidebarFacilitiesTab({
                                                 />
                                             </Link>
                                             <FeatureFlag flag={REPORT_A_FACILITY}>
-                                                {isClosed && (
+                                                {isClosed ? (
                                                     <div style={facilitiesTabStyles.closureRibbon}>
                                                         Closed facility
                                                     </div>
-                                                )}
+                                                ) : <div />}
```

Comment:
The feature flag component requires a child component, so returning `null` here created a propTypes error. I could instead move the isClosed check outside of the FeatureFlag component, so the FeatureFlag itself isn't rendered if the facility is open - I went with this route because it felt a bit more logical to me to say that we only care if the facility is closed if the feature flag is active.